src/libraries/System.Security.Cryptography.X509Certificates/tests/CollectionTests.cs

```diff
@@ -1563,6 +1563,56 @@ public static void ExportPkcs7_Empty()
             Assert.Equal("1.2.840.113549.1.7.2", oid); //signedData (PKCS #7)
         }
 
+        [Fact]
+        public static void TryExportCertificatePems_Empty()
+        {
+            X509Certificate2Collection cc = new X509Certificate2Collection();
+           AssertPemExport(cc, string.Empty);
+        }
+
+        [Fact]
+        public static void ExportCertificatePems_SingleCert()
+        {
+            const string SinglePem = "-----BEGIN CERTIFICATE-----\n" +
+                "MIIBETCBuaADAgECAgkA9StU5ZnBmM4wCgYIKoZIzj0EAwIwDzENMAsGA1UEAxME\n" +
+                "dGlueTAeFw0yMTA5MTUyMjAyNDNaFw0yMTA5MTUyMjAyNDNaMA8xDTALBgNVBAMT\n" +
+                "BHRpbnkwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNCAAQZ+baUXzzLi+p3cZEf4f23\n" +
+                "L/2Dbn5UB/uMCB7L71rWf3UwuCA3Is5uPci/3PQYLNwDkP3m3ZzxyzVCgFVqqYFg\n" +
+                "MAoGCCqGSM49BAMCA0cAMEQCIHafyKHQhv+03DaOJpuotD+jNu0Nc9pUI9OA8pUY\n" +
+                "3+qJAiBsqKjtc8LuGtUoqGvxLLQJwJ2QNY/qyEGtaImlqTYg5w==\n" +
+                "-----END CERTIFICATE-----";
+            using ImportedCollection ic = Cert.ImportFromPem(SinglePem);
```

Comment:
While the repository, collectively, is OK with braceless usings in test code, I'll make one plea to keep the braces :)

Reply:
Your house, your rules 😄.